README.md

```diff
@@ -1,36 +1,25 @@
 # datadog-api-client-python
 
-This repository contains a Python API client for the [Datadog API](https://docs.datadoghq.com/api/).
-The code is generated using [openapi-generator](https://github.com/OpenAPITools/openapi-generator)
-and [apigentools](https://github.com/DataDog/apigentools).
-
-Datadog API Collection
-
-- API version: 1.0
+This library allows you to quickly and easily use the [Datadog API](https://docs.datadoghq.com/api/) v1 and v2 with Python.
 
-Collection of all Datadog Public endpoints.
-
-  For more information, please visit [https://www.datadoghq.com/support/](https://www.datadoghq.com/support/)
-
-*Automatically generated by the [OpenAPI Generator](https://openapi-generator.tech)*
+The code is automatically generated using [openapi-generator](https://github.com/OpenAPITools/openapi-generator)
+and [apigentools](https://github.com/DataDog/apigentools).
 
 ## Requirements
 
-Building and using the API client library requires:
+Building and using the API client library requires [Python 3.6+](https://www.python.org/downloads/).
 
-1. Python 3.6+
+To use the Datadog API, you need Datadog [API](https://app.datadoghq.com/account/settings#api) and [Application keys](https://app.datadoghq.com/access/application-keys).
 
-## Installation
-
-To install the API client library, simply execute:
+## Install Package
 
 ```shell
 pip install datadog-api-client
 ```
 
-## Getting Started
+## Authentication
```

Comment:
I don't think the changes here works. This is not an `Authentication` example, this is a full example.